internal/bundlegen/generateapi.go

```diff
@@ -114,6 +114,9 @@ func LoadDocumentFromFile(filePath string, validate bool, formatValidation bool)
 	var err error
 	var jsonContent []byte
 
+	// see ./test/circular-reference.json and https://github.com/apigee/apigeecli/issues/199
+	openapi3.CircularReferenceCounter = 10
```

Comment:
Should be greater than 10, I suggest 20.

Also needs to be added to LoadDocumentFromURI